empire.go

```diff
@@ -708,6 +760,10 @@ func (e *Empire) StreamLogs(app *App, w io.Writer, duration time.Duration) error
 
 // CertsAttach attaches an SSL certificate to the app.
 func (e *Empire) CertsAttach(ctx context.Context, app *App, cert string) error {
+	if err := authorizeApp(ctx, "CertsAttach", app); err != nil {
```

Comment:
One thing I don't like right now, is there are inconsistently named methods: e.g. `CertsAtttach` and `DomainsCreate` vs `StreamLogs`. I think I prefer the later (`StreamLogs`, `AttachCert`, `AddDomain`, etc) as it reads a little nicer.

An argument could be made for the reverse, since it allows for `Prefix*` in ACL's (e.g. `Logs*`, `Domains*`).

I think, whatever we decide on, we should be consistent.

Thoughts? Preferences?